frontend/components/search/component.tsx

```diff
@@ -47,8 +47,9 @@ export const Search: FC<SearchProps> = ({
         ref={ref}
         placeholder={placeholder}
         type="search"
+        style={{ background: 'transparent' }}
         className={cx(
-          'w-full font-sans px-10 bg-transparent truncate leading-4 placeholder-gray-300 placeholder-opacity-50',
+          'w-full font-sans px-12 truncate leading-4 placeholder:text-gray-800 placeholder:text-base',
```

Comment:
To change the outline from blue to green

Reply:
I think the icon to clear the input is too small. It could be the same size as the search icon

Line 63: `<Icon icon={xIcon} className="inline-block w-5 h-5" />`

(I'm writing here because I can't comment the icon line)